mappings/net/minecraft/world/level/storage/LevelSummary.mapping

```diff
@@ -2,7 +2,7 @@ CLASS net/minecraft/class_34 net/minecraft/world/level/storage/LevelSummary
 	FIELD field_205 name Ljava/lang/String;
 	FIELD field_209 requiresConversion Z
 	FIELD field_23772 locked Z
-	FIELD field_23773 file Ljava/nio/file/Path;
+	FIELD field_23773 path Ljava/nio/file/Path;
```

Comment:
I think `iconFile` best describes this field. Can be fixed later though.

Reply:
I'll fix that one now.